src/index.html

```diff
@@ -65,7 +65,17 @@ <h1 class="header">Project 1</h1>
             <img src="https://via.placeholder.com/200x200" alt="project image">
             <img src="https://via.placeholder.com/200x200" alt="project image">
           </div>
-        </1div>
+          <hr size="4" align="center" color="#666262">
```

Comment:
`align="center"` is redundant. Why did you choose that color? The separator has a different color in figma.

Reply:
Also fix the syntax error at line 68 :P